mule-linter-core/src/main/groovy/com/avioconsulting/mule/linter/dsl/MuleLinterDsl.groovy

```diff
@@ -0,0 +1,102 @@
+package com.avioconsulting.mule.linter.dsl
+
+import com.avioconsulting.mule.linter.model.rule.RuleSet
+import com.avioconsulting.mule.linter.rule.cicd.*
+import com.avioconsulting.mule.linter.rule.configuration.*
+import com.avioconsulting.mule.linter.rule.pom.*
+import com.avioconsulting.mule.linter.rule.git.GitIgnoreRule
+
+import static groovy.lang.Closure.DELEGATE_ONLY
+
+
+class MuleLinterDsl {
+
+    RulesDsl rulesDsl
+
+    enum RULE {
+        JENKINS_EXISTS(JenkinsFileExistsRule.class),
+        AZURE_PIPELINES_EXISTS(AzurePipelinesExistsRule.class),
+        GIT_IGNORE(GitIgnoreRule.class),
+        CONFIG_FILE_NAMING(ConfigFileNamingRule.class),
+        FLOW_SUBFLOW_NAMING(FlowSubflowNamingRule.class),
+        MUNIT_VERSION(MunitVersionRule.class)
```

Comment:
Are we going with static list of rules? I think this should be discovered instead.

Reply:
Yes, but that feauture will be cover in other branch autodiscovery-rules, because @adesjardin  mentioned that in this one not, but the idea is how you mentioned @manikmagar need to be discovered